docs/deploying.md

```diff
@@ -2,155 +2,135 @@
 
-
 # Deploying a registry server
 
-You obviously need to [install Docker](https://docs.docker.com/installation/) (remember you need **Docker version 1.6.0 or newer**).
+You need to [install Docker version 1.6.0 or newer](https://docs.docker.com/installation/).
 
-## Getting started
+## Running on localhost
 
 Start your registry:
 
-     $ docker run -d -p 5000:5000 \
-     		--restart=always --name registry registry:2
+    docker run -d -p 5000:5000 --restart=always --name registry registry:2
 
-That's it.
+You can now use it with docker.
 
-You can now tag an image and push it:
+Get any image from the hub and tag it to point to your registry:
 
-    $ docker pull ubuntu && docker tag ubuntu localhost:5000/batman/ubuntu
-    $ docker push localhost:5000/batman/ubuntu
+    docker pull ubuntu && docker tag ubuntu localhost:5000/ubuntu
 
-Then pull it back:
+... then push it to your registry:
 
-    $ docker pull localhost:5000/batman/ubuntu
+    docker push localhost:5000/ubuntu
 
-## Where is my data?
+... then pull it back from your registry:
 
-By default, your registry stores its data on the local filesystem, inside the container.
+    docker pull localhost:5000/ubuntu
 
-In a production environment, it's highly recommended to use [another storage backend](storagedrivers.md), by [configuring it](configuration.md).
+Now, stop your registry:
 
-If you want to stick with the local posix filesystem, you should store your data outside of the container.
+    docker stop registry && docker rm -v registry
 
-This is achieved by mounting a volume into the container:
+## Storage
 
-     $ docker run -d -p 5000:5000 \
-        -e REGISTRY_STORAGE_FILESYSTEM_ROOTDIRECTORY=/var/lib/registry \
-        -v /myregistrydata:/var/lib/registry \
-        --restart=always --name registry registry:2
+By default, your registry data is persisted as a [docker volume](https://docs.docker.com/userguide/dockervolumes/) on the host filesystem. Properly understanding volumes is essential if you want to stick with a local filesystem storage.
 
-## Making your Registry available
+Specifically, you might want to point your volume location to a specific place in order to more easily access your registry data. To do so you can:
 
-Now that your registry works on `localhost`, you probably want to make it available as well to other hosts.
+	docker run -d -p 5000:5000 --restart=always --name registry \
+        -v `pwd`/data:/var/lib/registry \
+        registry:2
 
-Let assume your registry is accessible via the domain name `myregistrydomain.com` (still on port `5000`).
+### Alternatives
 
-If you try to `docker pull myregistrydomain.com:5000/batman/ubuntu`, you will see the following error message:
+You should usually consider using [another storage backend](https://github.com/docker/distribution/blob/master/docs/storagedrivers.md) instead of the local filesystem, by [configuring it](https://github.com/docker/distribution/blob/master/docs/configuration.md#storage).
```

Comment:
How about:

    You should usually consider using [another storage backend](https://github.com/docker/distribution/blob/master/docs/storagedrivers.md) instead of the local filesystem. Use the [storage configuration options](https://github.com/docker/distribution/blob/master/docs/configuration.md#storage) to configure an alternate storage backend.